meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
`#hintworthy_node?` checks if the default is a hash. I could have added an entirely new example but since `baz` was an arbitrary placeholder it seemed appropriate to substitute a hash.

tl;dr: it kills the mutation.